src/data.js

```diff
@@ -2558,6 +2558,27 @@ module.exports = [
       'GraphQL',
     ],
   },
+  {
+    name: 'Zoran Panev',
+    description: 'Web developer',
+    url: 'https://gist.github.com/8f08557764711afbf82b75ac0ce61e79.git',
```

Comment:
You're gonna need to actually put the gear/software you use in this gist

Reply:
> You're gonna need to actually put the gear/software you use in this gist

Did you mean something like this? Is it now fine? I have updated my gist.